profiles/package.mask

```diff
@@ -458,7 +467,9 @@ ros-meta/audio_common
 ros-meta/bond_core
 ros-meta/calibration
 ros-meta/common_msgs
-ros-meta/common_tutorials
+tps://github.com/gentoo/gentoo/commit/e7445024a6f389f74665c927d95c2274e8cf43b9), please make the commit message's first line/title/summary descriptive of what you're actually fixing ("add unconditional gtest dep").
```

Comment:
this looks like commit damage that needs fixing

Reply:
I have no idea on how it happened, fixed now